### PR TITLE
docs/3590/added-chart-argument

### PR DESCRIPTION
**Pull request by ppotaczek**
Added chart argument description in Highcharts.error method.

### CODE REVIEW

File: js/error-messages.js [hunk 1]
```diff
@@ -124,6 +124,6 @@ H.errorMessages = {
             "errors/29/readme.md",
             "errors/30/readme.md"
         ],
-        "version": "6.2.0"
```
Comment (KacperMadej) [inline js/error-messages.js]: This change is against the "DO NOT EDIT!" rule placed in this file.
https://github.com/highcharts/highcharts/blob/ec1b1667a800dcf9eead053b89debc21cdeb772c/js/error-messages.js#L8

@bre1470 btw. Could you take a look? Running `gulp` updates this file and the file's current content on the remote seems not up-to-date. How and when will be this file updated?

**Comment by bre1470** [comment]
This comes from `./tools/error-messages.js`
```js
            let parsedErrors = {
                meta: {
                    files: [],
                    version: require(rootPath  + '/package.json').version
                }
            };
```
Is this wrong, @TorsteinHonsi ?

**Comment by bre1470** [comment]
Oh, I realize now the problem. ErrorMessages rewrites the JSON files only, if the makrdown files change. I could add, that changes the JSON file also when the package version has changed. Should I add this? Or should version get removed?

**Comment by TorsteinHonsi** [comment]
Maybe just remove the version? I don't think we need it - if someone requires a specific version it could be loaded directly from a specific GitHub tag or from `code.highcharts.com/{version}`?

**Comment by bre1470** [comment]
If this is fine with Highcharts Cloud, I will just remove it. Objections, @cvasseng and @kai2468 ?

**Pull request merged**